app/elements/youneeq-tracker/youneeq-tracker.js

```diff
@@ -134,7 +134,7 @@ class youneeqTracker {
   }
 
   _contentChanged(content, oldContent) {
-    if (typeof content !== 'undefined' && Object.keys(content).length > 0) {
+    if (typeof content !== 'undefined' && Object.keys(content).length > 0 && !this.hidden) {
```

Comment:
I see, where did you intend them to go when you suggested adding them "the gallery and story pages are in the DOM and therefore are listening for the events. So the send page hit and send observe methods should be wrapped in a simple if (!this.hidden)"?